src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -484,6 +484,8 @@ protected override bool FindFirstChar()
                         }
                     }
                 }
+
+                runtextpos = runtextend;
```

Comment:
How did you find these changes are needed? Is there any possible way of adding a test to guard it?

Reply:
I just happened to notice it was missing as compared to what had previously been there. Functionally, I don't think it matters; if FindFirstChar returns false in https://github.com/dotnet/runtime/blob/20ab2fa963934b5823e0b8afbba858169dd27169/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunner.cs#L140, then we bump the current position by 1 and loop around to try again.  The issue would be performance, if we could have skipped more characters and are instead to repeatedly iterate by 1 instead of something larger.  I'm not sure of a good example to trigger it, though.